metadata/index.html

```diff
@@ -1446,6 +1459,7 @@
           <li>when <strong>displaying</strong> data, to map it into formats that are meaningful for those viewing the data (as opposed to those publishing it)</li>
           <li>when <strong>inputting</strong> data, to turn entered values into representations in a consistent format</li>
         </ul>
+        <p>The <dfn>cell value</dfn> of a cell is the result processing the cell as defined here and applying any format-specific mappings to create on or more values. Converters MUST be provided these cell values, along with any in-scope <code>lang</code> and <code>datatype</code> <a title="property value">property values</a>.</p>
```

Comment:
Isn't the value of a cell defined within the model document? Why not reference that?

s/result processing/result of processing/

Aren't converters provided everything from the metadata?